recipe/run_test.sh

```diff
@@ -1,12 +1,6 @@
 #!/bin/bash
 set -ex
 
-export OMPI_MCA_pml=ob1
-export OMPI_MCA_btl=sm,self
-export OMPI_MCA_plm_ssh_agent=false
-export OMPI_MCA_rmaps_default_mapping_policy=:oversubscribe
-export OMPI_ALLOW_RUN_AS_ROOT=1
```

Comment:
@minrk Are you absolutely sure `mpiexec` will never be invoked by the root user?

Reply:
I don't believe any commands inside recipes are invoked as root in conda-forge at least, which is my main goal to cover here. Additional builds elsewhere are still free to specify additional arguments if they need them, as they have always needed to do before this.